### ISSUE TITLE
Maybe.fromEmpty is not yet added to src/monet.d.ts

### ISSUE THREAD

**Issue by xanonid** [monet/monet.js#228]
Maybe.fromEmpty is implemented, but not yet listed in src/monet.d.ts.

**Comment by doktor500** [comment]
Hi, could this be added and released?

**Comment by ulfryk** [comment]
@doktor500 I'm not sure when I'll have time to do it.. and there's not much more people here that try to maintain this library 🙈

**Comment by tomecko** [comment]
I can prepare release of 0.9.3 if the above change gets merged.

**Comment by ulfryk** [comment]
@tomecko done :)

**Comment by tomecko** [comment]
I believe I've done what was possible for me to do. The rest lies in the hands of maintainer @ulfryk

**Comment by doktor500** [comment]
Great, @ulfryk would you mind releasing this version to npm?

**Comment by ulfryk** [comment]
@doktor500 @tomecko it should have automatically release after merge :(

I don't have time to investigate now, sorry

**Comment by ulfryk** [comment]
but snapshot with this functionality was automatically released with version 0.9.2-479

**Comment by doktor500** [comment]
It seems that the changes were merged to the `develop` branch but they still need to be merged to the `master` branch in order to create a version on npm with the right tag (`0.9.3`) would you mind merging those changes to `master` and see if that automatically creates the new release on npm with the right version tag?

**Comment by doktor500** [comment]
Basically merging this PR https://github.com/monet/monet.js/pull/248

**Comment by ulfryk** [comment]
Exactly, thanks, sorry for all delays:

```
A new version of the package monet (0.9.3) was published at 2021-11-15T18:04:01.737Z from
34.66.200.49. The shasum of this package was b296747d5dc6dc13f51b4c7e85ed01d787b031a1.
```

🎉

**Comment by doktor500** [comment]
Brilliant! Thanks so much, @ulfryk and @tomecko 👏
